adafruit_minimqtt/matcher.py

```diff
@@ -11,6 +11,13 @@
 * Author(s): Yoch (https://github.com/yoch)
 """
 
+try:
+    from typing import Any, Dict
+except ImportError:
+    pass
+
+from collections.abc import Callable, Iterator
```

Comment:
CircuitPython doesn't support `collections.abc`, so this should go after the `typing` import.  Additionally, it looks like there was a bug with using `collections.abc.Callable` for typing in 3.9.1, so I would just use `typing.Callable` and `typing.Iterator`.

Reply:
okay, should be fixed.